odd_collector/adapters/postgresql/mappers/tables.py

```diff
@@ -57,56 +55,66 @@ def map_table(
                 oddrn=oddrn_generator.get_oddrn_by_path(oddrn_path),
                 name=table_name,
                 type=data_entity_type,
-                owner=metadata.table_owner,
-                description=metadata.description,
+                owner=table.table_owner,
+                description=table.description,
                 metadata=[],
             )
             data_entities.append(data_entity)
 
             append_metadata_extension(
                 data_entity.metadata,
                 _data_set_metadata_schema_url,
-                metadata,
+                table,
                 _data_set_metadata_excluded_keys,
             )
 
             data_entity.dataset = DataSet(
-                rows_number=int(metadata.table_rows)
-                if metadata.table_rows is not None
+                rows_number=int(table.table_rows)
+                if table.table_rows is not None
                 else None,
                 field_list=[],
             )
 
             # DataTransformer
             if data_entity_type == DataEntityType.VIEW:
                 data_entity.data_transformer = extract_transformer_data(
-                    metadata.view_definition, oddrn_generator
+                    table.view_definition, oddrn_generator
                 )
 
+            etl_grouped = {}
```

Comment:
You can use here `defaultdict(list)` and do not use `if etl.type_oid not in etl_grouped:` below.